v4/newrelic/segments.go

```diff
@@ -301,6 +313,9 @@ func (s *MessageProducerSegment) name() string {
 	if s.DestinationTemporary {
 		dest = "(temporary)"
 	}
+	if dest == "" {
+		dest = "Unknown"
+	}
```

Comment:
#### Bug 3

Based on a `MessageBroker` metric, I realized that we should be setting the destination to `Unknown` when it is unknown.